project/core/templates/landing.html

```diff
@@ -1,172 +1,183 @@
-                            <br>
-                            <br> {% trans "Second, CiviWiki will collect anonymized user data and forward district-level statistics to representatives. With a critical mass of users, we believe timely district level polling data will influence representatives votes." %}
-                        </p>
-                    </div>
-
-                </div>
-            </div>
-        </div>
-
-    </div>
-    <div class="lastcall cushion">
-        <div class="container section">
-            <div class="section">
-                <h5 class="center text-call">{% trans "See what your voice can bring to your community" %}</h5>
-            </div>
-            <div class="section row center">
-                <a href="login" class="btn btn-large waves-effect waves-light"><span class="dark-grey-text">{% trans "Join CiviWiki" %} </span></a>
-            </div>
-        </div>
-    </div>
-    <div class="in-development">
-        <div class="container section center">
-            <h3 class="center"><i class="small material-icons yellow-text text-darken-2">warning</i> CiviWiki {% trans "is still in" %} <span class="text-highlight">{% trans "development" %}</span>! <i class="small material-icons yellow-text text-darken-2"></i></h3>
-            <div class="row">
-                <div class="container section center">
-                    <div class="col s12 m5 center">
-                        <a class="btn btn-large" href="login">{% trans "Register" %}</a>
-                    </div>
-                    <div class="col s12 m2">
-                        <h4 class="center ampersand">&</h4>
-                    </div>
-                    <div class="col s12 m5 center">
-                        <a class="btn btn-large " href="support_us">{% trans "Support Us" %}</a>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </div>
-</div>
-</main>
-
-</script>
-{% endblock backbone_template %}
-
-{% block content %}
-{% include "static_nav.html" %}
-    <div id="landing"></div>
-{% endblock content %}
+                    </span></td>
+                    <td>
+                        <h2 class="subtitle">Democratically Contributed Media</h2>
+                        <div class="text">
+                            As the name CiviWiki implies, our core content will be contributed by volunteers on our
+                            Wiki.<br><br>
+                            Our topic format is modular.<br><br>
+                            The structure both allows a community of volunteers to collaborate on a single political issue,
+                            and reserves space for dissenting opinions.
```

Comment:
Let's use `p` tags and the `translate` template tag here.
```suggestion
                        <div class="text">
                            <p>{% translate "As the name CiviWiki implies, our core content will be contributed by volunteers on our Wiki." %}</p>
                            <p>{% translate "Our topic format is modular." %}</p>
                            <p> {% translate "The structure allows a community of volunteers to collaborate on a single political issue and reserves space for dissenting opinions." %}</p>
```